### PR TITLE
Add support for CircleCI

### LINKED ISSUE #9: Support CircleCI.
My project(s) uses circle ci for our tests and was I was wondering if you could support that soon.

### PR DESCRIPTION
**Pull request by JuanitoFatas**
This pull request adds support for CircleCI and regenerated db/schema.rb.

fixes #9
## 

[CircleCI API doc: Recent Builds for a Single Project](https://circleci.com/docs/api#recent-builds-project)

### CODE REVIEW

File: config/locales/simple_form.en.yml [hunk 1]
```diff
@@ -61,4 +61,5 @@ en:
         repo_name: E.g. rails/rails, jollygoodcode/twemoji
         travis_token: Retrieve Travis Token from the Profile tab on your Accounts page
         codeship_uuid: Retrieve Project UUID from https://codeship.com/projects/[PROJECTID]/configure
+        circleci_token: "Add an CircleCI Token your account dashboard. https://circleci.com/docs/api#authentication"
```
Comment (winston) [inline config/locales/simple_form.en.yml]: Add `a` CircleCI.

Comment (winston) [inline config/locales/simple_form.en.yml]: I think we should also add to the hint >> `(required for private repos only)`. wdyt?

**Comment by winston** [comment]
LGTM! Thanks!

File: config/locales/simple_form.en.yml (hunk 1 above)
Comment (JuanitoFatas) [inline config/locales/simple_form.en.yml]: Should be a paragraph before all the token fields?

Comment (winston) [inline config/locales/simple_form.en.yml]: Can. i.e.:

`Required for private repos. Retrieve Travis Token from the....`

Comment (JuanitoFatas) [inline config/locales/simple_form.en.yml]: Added in 28dfbd3

screenshot 2015-10-07 01 20 40

**Comment by winston** [comment]
:heart: :bow:

**Pull request merged**